Caboodle/Permissions/PermissionEnums.shared.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Microsoft.Caboodle
+{
+    internal enum PermissionStatus
+    {
+        // Denied by user
+        Denied,
+
+        // Feature is disabled on device
+        Disabled,
+
+        // Granted by user
+        Granted,
+
+        // Restricted (only iOS)
+        Restricted,
+
+        // Permission is in an unknown state
+        Unknown
+    }
+
+    internal enum PermissionType
+    {
+        Unknown,
```

Comment:
Should we have an unknown?

Reply:
There's a few spots in iOS and one in UWP where we return this....